roles/pulp_devel/tasks/galaxy_ui.yml

```diff
@@ -1,69 +1,70 @@
 ---
-- name: Install NPM (distro-agnostic)
-  package:
-    name: npm
-    state: present
-  retries: "{{ pulp_devel_package_retries }}"
-  register: result
-  until: result is succeeded
+- name: Install galaxy UI from source
+  block:
+    - name: Install NPM (distro-agnostic)
+      package:
+        name: npm
+        state: present
+      retries: "{{ pulp_devel_package_retries }}"
+      register: result
+      until: result is succeeded
 
-- name: Check if UI is installed
-  stat:
-    path: '{{ developer_user_home }}/devel/ansible-hub-ui'
-  register: is_hub_installed
+    - name: Check if UI is installed
+      stat:
+        path: '{{ developer_user_home }}/devel/ansible-hub-ui'
+      register: is_hub_installed
 
-- name: UI from devel path
-  set_fact:
-    hub_ui_path: '{{ is_hub_installed.stat.path }}'
-  when: is_hub_installed.stat.exists
+    - name: UI from devel path
+      set_fact:
+        hub_ui_path: '{{ is_hub_installed.stat.path }}'
+      when: is_hub_installed.stat.exists
```

Comment:
Please use spaces rather than tabs.

Reply:
I think these are spaces. I was just increasing the indent to put those tasks in a block.

Reply:
Oh, gotcha. My bad.